### DISCUSSION TITLE
Body as content container

### DISCUSSION THREAD

**Discussion by rohansingh20** [swup/swup#371] [Help] [answered]
Im just started working with SWUP, and am having a couple issues, 

I am having some of my HTML not load in, and have to refresh each time. This was not happening before I used SWUP.

Also, do I have to use the main tags in swup or can I add the `id="swup" class="transition-fade"` into the body tag?

Thanks

**Comment by daun** [comment, marked as answer]
In case somebody is reading this later — swup containers must be children of the body to animate properly.
